frontend/src/types/global.d.ts

```diff
@@ -10,3 +10,10 @@ declare module "*.svg" {
   // eslint-disable-next-line import/no-default-export -- deprecated usage
   export default content;
 }
+
+// This allows importing CSS from TypeScript files
+declare module "*.css" {
```

Comment:
Allow importing CSS in TypeScript files since I converted some files that uses CSS modules to TypeScript